### ISSUE TITLE
build failure on OCaml 4.04

### ISSUE THREAD

**Issue by gasche** [aantron/bisect_ppx#112]
opam-builder reports that `bisect_ppx` needs to be updated to work on OCaml 4.04: http://opam.ocamlpro.com/builder/html/bisect_ppx/bisect_ppx.1.1.0/7b6c8c5404fcbd94d5f63514ad51d9da

```
+ ocamlfind ocamlopt -c -package ppx_tools -w A-4 -warn-error A-3 -I src/syntax -I doc -I src -I src/report -I src/ocamlbuild -I src/library -o src/syntax/instrumentPpx.cmx src/syntax/instrumentPpx.ml
File "src/syntax/instrumentPpx.ml", line 152, characters 4-2061:
Warning 8: this pattern-matching is not exhaustive.
Here is an example of a case that is not matched:
Ppat_open (_, _)
File "src/syntax/instrumentPpx.ml", line 1:
Error: Some fatal warnings were triggered (1 occurrences)
Command exited with code 2.
```

**Comment by rleonid** [comment]
@gasche Can you please provide some guidance on this new pattern `Ppat_open`. I haven't followed all of the forthcoming changes in 4.04.

**Comment by rleonid** [comment]
Ah found it: https://github.com/ocaml/ocaml/pull/578.

**Comment by aantron** [comment]
Fixed in #113.
